src/robot_description/models/master_ASM/urdf/master_ASM.urdf.xacro

```diff
@@ -5,6 +5,14 @@ Stephen Brawner (brawner@gmail.com)
      For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
 
 <robot name="Master_ASM" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <link name="root_link">

```

Comment:
base_link is standard why use root?

Reply:
I know I had a reason... but now I forget what it was, lol. I think it was something to do with you choosing frame_link instead of base_link previously. There hasn't been a base_link in our robot urdf for a long time.

Reply:
I just fixed this by renaming "root_link" to "base_link" and everything still works